site/docs/spark-queries.md

```diff
@@ -191,20 +190,19 @@ join prod.db.table.snapshots s
   on h.snapshot_id = s.snapshot_id
 order by made_current_at
 ```
-```text
-+-------------------------+-----------+----------------+---------------------+----------------------------------+
-| made_current_at         | operation | snapshot_id    | is_current_ancestor | summary[spark.app.id]            |
-+-------------------------+-----------+----------------+---------------------+----------------------------------+
+
+<div class="markdown-table-container" markdown="block">
+| made_current_at | operation | snapshot_id | is_current_ancestor | summary[spark.app.id] |
+| -- | -- | -- | -- | -- |
 | 2019-02-08 03:29:51.215 | append    | 57897183625154 | true                | application_1520379288616_155055 |
 | 2019-02-09 16:24:30.13  | delete    | 29641004024753 | false               | application_1520379288616_151109 |
 | 2019-02-09 16:32:47.336 | append    | 57897183625154 | true                | application_1520379288616_155055 |
 | 2019-02-08 03:47:55.948 | overwrite | 51792995261850 | true                | application_1520379288616_152431 |
-+-------------------------+-----------+----------------+---------------------+----------------------------------+
-```
+</div>
 
 ### Files
 
-To show a table's data files and each file's metadata, run:
+To show a table's current data files and each file's metadata, run:
```

Comment:
I honestly don't think 'each file's metadata' brings much value , but I guess it's already there.